content/test/integration/service/deployments/old-entity-synced.spec.ts

```diff
@@ -0,0 +1,88 @@
+import { AuthLinkType, Entity, EntityType } from "@dcl/schemas";
+import { AuditInfo } from "dcl-catalyst-commons";
+import { createFsComponent } from '../../../../src/ports/fs';
+import { DeploymentContext, DeploymentResult } from "../../../../src/service/Service";
+import { AppComponents } from "../../../../src/types";
+import { makeNoopServerValidator } from "../../../helpers/service/validations/NoOpValidator";
+import { loadStandaloneTestEnvironment, testCaseWithComponents } from "../../E2ETestEnvironment";
+import { EntityCombo } from "../../E2ETestUtils";
+import { getIntegrationResourcePathFor } from '../../resources/get-resource-path';
+
+const fs = createFsComponent()
+
+loadStandaloneTestEnvironment()("Integration - Deployment synced old entity", (testEnv) => {
+
+  testCaseWithComponents(
+    testEnv,
+    "When deploying an old synced entity, it should succeed",
```

Comment:
Could we explicitly say that it's an entity with `cid v0` hashes of entities before ADR 45? Or just that it's before ADR 45?